src-java/kilda-pce/src/main/java/org/openkilda/pce/finder/FailReasonType.java

```diff
@@ -0,0 +1,36 @@
+/* Copyright 2022 Telstra Open Source
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package org.openkilda.pce.finder;
+
+public enum FailReasonType {
+    NO_CONNECTION("There is no connection"),
+    ALLOWED_DEPTH_EXCEEDED("Allowed depth exceeded"),
+    LATENCY_LIMIT("Latency Limit"),
```

Comment:
In all other reasons only first word starts with capital character. please use `limit`

Reply:
done